log/roller.go

```diff
@@ -88,19 +90,22 @@ func (l Roller) GetLogWriter() io.Writer {
 	if err != nil {
 		absPath = l.Filename // oh well, hopefully they're consistent in how they specify the filename
 	}
+
```

Comment:
pls fix comment on  `GetLogWriter`.
```
// GetLogWriter returns an io.Writer that writes to a rolling logger.
// This should be called only from the main goroutine (like during
// server setup) because this method is not thread-safe
```